payme.php

```diff
@@ -167,6 +167,27 @@ public function callback()
                 $this->respond($this->error_invalid_json());
             }
 
+			/** 
+				Fix support php-fpm
+			**/
+			
+			
+			if (!function_exists('getallheaders'))
```

Comment:
Can you move this implementation before woocommerce_payme() (to line 15), please.
Please, also follow our formatting style, I mean, 4 spaces instead of tabs.
Than I can merge this commit.
Thank you for your time to improve our plugin!!!